windows-oci-context/README.md

```diff
@@ -0,0 +1,65 @@
+# Using an OCI context
```

Comment:
Could we maybe add a note on when using this might be beneficial. As far as I can tell this is the first mention of using an OCI context in our docs so it might be good to give some additional guidance and context to users